### DISCUSSION TITLE
feedback/showcase: Dracula inspired Leonids theme

### DISCUSSION THREAD

**Discussion by bigpick** [dracula/dracula-theme#409] [General]
## Background
Hi there! Just as soon as I finished looking at [the first Dracula community](https://community.draculatheme.com/) I heard this was the place instead, so posting this here.

Long time Dracula user, and hopefully soon to be contributor (looking for projects). I wanted to share with you all my personal page, which is a Dracula inspired take on the [Leonids](https://github.com/renyuanz/leonids/) Jekyll theme.

You can see the site [here](https://bigpick.github.io/TodayILearned/).

## Questions/thoughts
I'm curious what are thoughts on readability, and overall proper representation of the Dracula theme? In general, but at least specifically for
* the Cyan accent on the main page (date, author)
* link highlighting as green (wasn't sure if that is covered by [this spec listing](https://spec.draculatheme.com/#MarkupLinkUrl)?)

I just found out about the [Dracula specification](https://spec.draculatheme.com/), though that is tailored towards syntax highlighting (which for the site is done using the [Dracula Pygments theme](https://github.com/dracula/pygments))

I know there is a [Github pages theme](https://github.com/dracula/gh-pages) already, but I really prefer the initial layout that the Leonids them provides.

It's all pretty hackily done by just changing values inplace in css/scss files. If interested, I could try to clean it up into a single separate file + option, but I'm no front end developer so might take some time :)

**Comment by benknoble** [comment]
Random thought after a quick browse: some of the code snippets (e.g., the scala intro post) use a dark main color on a dark background... not easy to read.

**Reply by bigpick** [reply]
Hey, thanks for the look! That's something I actually went back and forth on originally. 

It's the syntax for generic or non-specifically highlighted code. Originally I had it as `#f8f8f2` (foreground) on `#282a36` (background), as that would match the `code` syntax which is the same, and also and seem to be in alignment with the [Pygments values](https://github.com/dracula/pygments/blob/314da2f3ef70653412928f50c9d3ed124c5c6309/dracula.css#L16); it seemed like a lot of foreground text on pages with not a lot of specific code, but just general code-block formatted sections, though.

I'm not particularly tied to the current `#6272a4` (comment) on background for generic, but was trying to pick something nice looking but still easier to read.
